x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/components/actions_menu.tsx

```diff
@@ -76,7 +76,7 @@ export const AgentPolicyActionMenu = memo<{
             ? [viewPolicyItem]
             : [
                 <EuiContextMenuItem
-                  disabled={!hasWriteCapabilities}
+                  disabled={!hasWritePermissions}
```

Comment:
This should not be disabled in any case now. We decided any users who can see this UI should be able to enroll agents.